src/containers/pages/FocusInvestigation/active/tests/index.test.tsx

```diff
@@ -1,5 +1,6 @@
 import { library } from '@fortawesome/fontawesome-svg-core';
 import { faExternalLinkSquareAlt } from '@fortawesome/free-solid-svg-icons';
+import DrillDownTable from '@onaio/drill-down-table/dist/types';
```

Comment:
Could you fix the linting errors in this file please?

Reply:
I think these are fixed already. some times VScode shows linting errors where there aren't if you close the file and open it again. They'll be gon

Reply:
I think this needs to be imported from somewhere else e.g. `import DrillDownTable from '@onaio/drill-down-table` i.e. dont import from `dist/types`